### ISSUE TITLE
Desktop application

### ISSUE THREAD

**Issue by johntdyer** [mashlol/notify#7]
Would be nice if there were an agent for the desktop as well.  Figured I would open a ticket to track that request.  Thanks

-John

**Comment by mashlol** [comment]
Thanks for the request! Could you clarify what the features might be of this desktop app?  Would it just be a giant text field where you could enter the notification message, and then a button to send it?  What's the use-case for that?

**Comment by johntdyer** [comment]
No, I was thinking more along the lines of receiving alerts on my desktop, not sending them

**Comment by mashlol** [comment]
Ah.  I don't think a desktop app would really be required for that, we could just use something like https://github.com/mikaelbr/node-notifier to push OS-level notifications at the same time as the cloud-based notifications go out.

**Comment by yvan-sraka** [comment]
I could be pretty interest in helping develop this Desktop version! 😄

**Comment by mashlol** [comment]
> I could be pretty interest in helping develop this Desktop version! 

Sure, happy to have your help. I was thinking of making a site that uses web push notifications and having that be the "register" part. Firebase can already push notifications to web. So then it would work on any platform (mac, linux, windows, etc)

**Comment by cheshirekow** [comment]
I was looking for this also, which is how I ended up finding your app. Ideally the desktop app would receive notifications the same way the phone would, so that I can see the notification on my phone, or whichever computer I'm actually sitting at. In other words I'm looking for more than just local notification. Currently I use `ssh ... notify-send` which only works within my local network.
